Competition/src/test/java/competition/subsystems/pose/PoseTest.java

```diff
@@ -0,0 +1,95 @@
+package competition.subsystems.pose;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+
+import xbot.common.math.ContiguousHeading;
+import xbot.common.math.FieldPose;
+
+import xbot.common.math.XYPair;
+import xbot.common.subsystems.pose.PoseSubsystemTest;
+
+public class PoseTest {
+    @Test
+    public void testFlipFieldPose() {
+        
+        FieldPose right1 = new FieldPose(new XYPair(324, 10), new ContiguousHeading(180));
+        FieldPose leftExpected1 = new FieldPose(new XYPair(0, 10), new ContiguousHeading(0));
+
+        assertEquals(leftExpected1.getPoint().x, PoseSubsystem.flipFieldPose(right1).getPoint().x, 0);  
+        assertEquals(leftExpected1.getPoint().y, PoseSubsystem.flipFieldPose(right1).getPoint().y, 0);
+        assertEquals(leftExpected1.getHeading().getValue(), PoseSubsystem.flipFieldPose(right1).getHeading().getValue(), 0);     
+    }
+    @Test
+    public void testFlipFieldPose2() {
```

Comment:
Rather than naming these Pose, Pose2, Pose3 etc.. can you give them more descriptive names about what the points are?